resources/load-test/load-test-participant.js

```diff
@@ -232,6 +232,8 @@ JitsiMeetJS.setLogLevel(JitsiMeetJS.logLevels.ERROR);
 
 JitsiMeetJS.init(config);
 
+config.serviceUrl = config.bosh = `${config.websocket || config.bosh}?room=${roomName.toLowerCase()}`;
```

Comment:
If you're gonna do `toLowerCase` on `roomName` (which I agree is probably a good idea, even we don't do the full canonicalization of internationalized room names) you should do it on the value that's passed to `initJitsiConference` as well.